Data Structures - Linked Lists/Linked List.py

```diff
@@ -61,7 +61,7 @@ def remove(self,index):
         self.head = temp.next
         self.length -= 1   
         break       
-      if i == self.length-1:
```

Comment:
I Implemented the remove function badly, can you please try to implement a more readable function

Reply:
i changed it in a new commit